semantics/language/translation/decl/local.k

```diff
@@ -179,27 +169,25 @@ module C-DECL-LOCAL
           ~> defineAndInit(_, _, initValue(_, T:Type, _))
           requires notBool isCompleteType(T)
           [structural]
+     rule <k> (.K => CV("TDL6","redeclaration of " +String idToString(X)
```

Comment:
I think this should have the same error code as TDL4 since it has the same error message and refers to the same paragraph in the standard.